pkgs/development/libraries/vtk/generic.nix

```diff
@@ -0,0 +1,98 @@
+{ majorVersion, minorVersion, sourceSha256, patchesToFetch ? [] }:
+{ stdenv, lib, fetchurl, cmake, libGLU, libGL, libX11, xorgproto, libXt, libtiff
+, fetchpatch
+, enableQt ? false, wrapQtAppsHook, qtbase, qtx11extras, qttools
+, enablePython ? false, pythonInterpreter ? throw "vtk: Python support requested, but no python interpreter was given."
+# Darwin support
+, Cocoa, CoreServices, DiskArbitration, IOKit, CFNetwork, Security, GLUT, OpenGL
+, ApplicationServices, CoreText, IOSurface, ImageIO, xpc, libobjc
+}:
+
+let
+  inherit (lib) optionalString optionals optional;
+
+  pythonMajor = if pythonInterpreter.isPy3 then "3" else if pythonInterpreter.isPy2 then "2" else throw "vtk: vtk only supports python 2.x and 3.x";
+
+  maybeToPythonModule = if enablePython then pythonInterpreter.pkgs.toPythonModule else lib.id;
```

Comment:
What's the advantage of moving `toPythonModule` and thereby deviating from the common approach? I think its important that contributors to `python-packages.nix` see they need to use `toPythonModule` in case of bindings, to reduce the chance in forgetting to add that.

Reply:
Reducing the amount of repetition. Not a hill I'm going to die on though, undid this change :)